proto/proto-backplane-grpc/src/main/proto/deephaven/proto/object.proto

```diff
@@ -22,5 +82,45 @@ message FetchObjectRequest {
 message FetchObjectResponse {
     string type = 1;
     bytes data = 2;
-    repeated io.deephaven.proto.backplane.grpc.TypedTicket typed_export_id = 3;
+    repeated io.deephaven.proto.backplane.grpc.TypedTicket typed_export_ids = 3;
```

Comment:
this is technically not a wire format change, but trying to fix a typo that we have. It break compiled code, where someone is consuming the protoc generated jars and building their own impl, but I think in this context we should be safe, as only js clients use this at this time.